.ci/e2eTestingMacosDaily.groovy

```diff
@@ -35,6 +37,8 @@ pipeline {
     string(name: 'ELASTIC_AGENT_VERSION', defaultValue: '8.4.0-SNAPSHOT', description: 'SemVer version of the Elastic Agent to be used for the tests. You can use here the tag of your PR to test your changes')
     string(name: 'ELASTIC_STACK_VERSION', defaultValue: '8.4.0-SNAPSHOT', description: 'SemVer version of the stack to be used for the tests.')
     string(name: 'BEAT_VERSION', defaultValue: '8.4.0-SNAPSHOT', description: 'SemVer version of the Beat to be used for the tests. You can use here the tag of your PR to test your changes')
+    string(name: 'GITHUB_CHECK_REPO', defaultValue: 'elastic-agent', description: 'Name of the GitHub repo to be updated. Only modified if this build is triggered from another parent stream (i.e. Beats).')
```

Comment:
What's the reason for using a default value?

```suggestion
    string(name: 'GITHUB_CHECK_REPO', defaultValue: '', description: 'Name of the GitHub repo to be updated. Only modified if this build is triggered from another parent stream (i.e. Beats).')
```

Reply:
It could be empty too, to be honest. We use the agent repo by default to be meaningful to the agent repo, but it would be totally OK as an empty string

Reply:
It might mislead, so I'd say to keep it empty